airbyte-webapp-e2e-tests/cypress/integration/connection/streamTable.spec.ts

```diff
@@ -103,6 +116,37 @@ describe("New stream table - new connection set up ", () => {
     selectSchedule("Manual");
   });
 
+  it("should check check connector icons and titles in table", () => {
+    checkConnectorIconAndTitle("source");
```

Comment:
Is this just checking that they have a value, but not what that value is?

Reply:
Looking below, yes. Not sure if it's worth confirming the icon and name are what we expect since that should be fed to us by the DB.

Reply:
We just check that icons and titles are displayed, but we don't check what exact icons are displayed.
Should I remove the test?

Reply:
Ehh, can't hurt. Well, I guess it can be another test in our already slow test suite but I don't think we should remove it.
Ignore my ramble :)